Modified 'user_login' view.

praneeth-rdy opened the pull request:
### Changed the code in 'user_login' view in the register django app.

**Previously:** When a someone tries to login and if the user doesn't exist -> Django returns rendered home.html
```python
return render(request, 'register/home.html', {'message': 'You need to register yourself first'})
```

**After Modification:** When a someone tries to login and if the user doesn't exist -> Django redirects the user to register page.
```python
return redirect('/register/')
```

Commit:
Modified user_login view. This modification redirects user to register page if the login credentials doesn't exist

ExpressHermes reviewed (commented):
Add the message "You need to register yourself first" When the user is redirected to the register page

Commit:
Added message 'You need to register yourself first' when login page redirects to the register page

praneeth-rdy commented:
Yeah! Now, I added that by using **Django inbuilt messages module**. Not as context.

```python
messages.warning(request, 'You need to register yourself first!')
return redirect('/register/')
```

Also modified base.html as to accept this django message and display it.
I added the following new snippet below the old message snippet without modifying it.
```django
{% for message in messages %}
        <div class="alert alert-info alert-dismissible fade show" role="alert"> {{ message }}
          <button type="button" class="close" data-dismiss="alert" aria-label="Close">
            <span aria-hidden="true">&times;</span>
          </button>
        </div>
{% endfor %}
```

**This won't create any issue with other pages using base.html**
